src/build.d

```diff
@@ -414,6 +433,33 @@ auto predefinedTargets(string[] targets)
     return newTargets.data;
 }
 
+// An input range for a recursive set of dependencies
```

Comment:
Please use a DDoc comment.  It makes the comment appear when hovering over a symbol in some developers' editors.

Reply:
done